pkg/controllers/disruption/metrics.go

```diff
@@ -53,24 +52,15 @@ var (
 		},
 		[]string{methodLabel, consolidationTypeLabel},
 	)
-	ActionsPerformedCounter = prometheus.NewCounterVec(
+	DecisionsPerformedCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: metrics.Namespace,
 			Subsystem: disruptionSubsystem,
-			Name:      "actions_performed_total",
-			Help:      "Number of disruption actions performed. Labeled by disruption action, method, and consolidation type.",
+			Name:      "decisions_total",
+			Help:      "Number of disruption decisions performed. Labeled by disruption type, method, and consolidation type.",
 		},
 		[]string{actionLabel, methodLabel, consolidationTypeLabel},
```

Comment:
do we need to update this label to be disruption type? Or should it remain as action? Should probably have that decision reflected in the description too. 
```suggestion
		[]string{disruptionTypeLabel, methodLabel, consolidationTypeLabel},
```

Reply:
I think that we should keep this as `actionLabel` and update the description.